### DISCUSSION TITLE
Best way to check contracts coverage

### DISCUSSION THREAD

**Discussion by clauBv23** [ConsenSys-archive/truffle#6151] [General Q&A]
Since solidity coverage is no longer supporting Truffle, what is the best way to check the contracts coverage in truffle projects?

**Comment by FlaviuWeb3** [comment]
HI @clauBv23 unfortunately there is no alternative for the Solidity coverage for truffle projects as of now.

**Reply by clauBv23** [reply]
Hi @FlaviuWeb3, thanks for your response. 
Is there any WIP project to solve it? 
Downgrading solidity-coverage to v0.7.0 (when truffle was supported) should work?
